notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -337,6 +337,44 @@ demon [Bill and Ted met a pair of rather friendly demons during their brief stay
 reconciliation [Bill and Ted patched things up with the Grim Reaper who'd had a falling out with the pair due to a lawsuit.]
 
 
+movie: Minor Premise (2020)
+===========================
+
+:: Title
+Minor Premise
+
+:: Description
+Attempting to surpass his father's legacy, a reclusive neuroscientist becomes
+entangled in his own experiment, pitting ten fragments of his consciousness
+against each other.
+
+:: Date
+2020-08-29
+
+:: References
+https://en.wikipedia.org/wiki/Minor_Premise_(film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+personal identity [Alli summed up the message of the story at the end by explaining how the individual emerges out of the combination of various different selves.]
```

Comment:
The film ends with this philosophical monologue:

"We're not merely computers, time stamping information our senses take in. Nor are we adding machines marking ones and zeroes on a cerebral ledger. There's something else. Traits that mark us, intelligence, anger, love, do not define us. For in isolation, these fragments are course, even unrecognizable. But when they coexist in a single consciousness, something emerges. Something material, yet immaterial. Dare I even say immeasurable. Something called self."

The current definition of **personal identity** asks the question of what precisely constitutes ones "personhood". Do We want to the theme to have the latitude to cover the content of such statements as above? My sense is that we'd do well to refashion it into a theme called something like **persistence of personhood** and then use it in such stories as [https://themeontology.org/story.php?name=ds93x13](ds93x13) where the viewer is made to ponder whether Bareil Antos is the same person after parts of his brain were replaced with artificial components. If we use it for situations like that, then I say take it out from this film.

Thoughts?

Reply:
A) I'm not against improving the name of a theme, but I don't think your suggestion is much of an improvement.
B) My understanding of the monologue that you quote is it asserts that humans are more than machines because of some undefinable quality. That's pretty much just the age old idea of "the soul" then, with some extra mambo-jumbo. We have elected to put "the soul" under organized religion because we don't subscribe to this belief ourselves, and it is usually wound up with religious nonsense. 

I suppose it is possible to discuss whether there is something soul-like, without involving the idea of deities. Invent a new metaphysics ("are humans more than machines"?) theme or use the parent theme directly? Or just use the religion theme "the soul".

Reply:
The **are humans more than machines** theme strikes me as a profitable theme to introduce, although it would be minor in this movie. Want me to add it as minor and add a proposal for its definition?

But let me take on on **personal identity** from another angle, we motivate its use in this story in the same way as we did for [https://themeontology.org/story.php?name=tos1x04](tos1x04); namely, "Was the real Ethan the composite of his nine alter egos or did any one alter ego have a claim to being the real Ethan?" Whatever is done, it seems to me that the **personal identity** usages are in need of a review. So my thinking here is to toss it in to this story and take stock of things thereafter. A list perhaps?

Reply:
I see my email reply did not get incorporated automagically this time:

Sure, I think are humans more than machines would be profitable too since it is a popular one, albeit rather incoherent since the constituent components of it are ill-defined. But we catalogue timetravel nonsense, so why not this?

If you want to use personal identity as it is that's fine too, but it should have a motivation that matches the definition. I guess "there was some nonsense about whether ??'s personal identity was more than the sum of the 9 different component personalities it had whimsically been split into".

Reply:
Re-motivated **personal identity** and added the to-be-defined in subsequent pull request **are humans more than machines** as a minor theme.

Reply:
I still don't much like your motivation because it sounds more like a psychological explanation than a metaphysical idea. You wrote earlier:

"Was the real Ethan the composite of his nine alter egos or did any one alter ego have a claim to being the real Ethan?"

That's a lot more on the mark imho.

Reply:
Lol. They are both satisfactory to me. I used a variation on your suggestion instead of the one I proposed to make you happy. But revised the motivation to read

"Was the real Ethan the composite of his nine alter egos or did any one alter ego have a claim to being the real Ethan?"

and pushed the change.

So, in the end, if you approve, I'll do a "squash and merge" and get this one out of the way.